solargeo/avg_el.pyx

```diff
@@ -0,0 +1,87 @@
+'''Average Elevation Angle'''
+
+import numpy as np
+import pandas as pd
+from pandas.tseries.frequencies import to_offset
+from solargeo import sunae
+
+
+def avg_el(time, lat, lon, tz=0, ref='BEG'):
+    '''
+    Calculates the average cosine solar zenith angle during the time interval
+    and returns the effective elevation angle. For instantaneous elevation
+    values call solargeo.sunae directly.
+
+    Converts timestamp/labels to beginning of bin.
+
+    Parameters
+    ----------
+    time : datetime index
+        assumed to be in UTC, conversion should occur outside of function
+    lat : float
+        site latitude (north is positive)
+    lon : float
+        site longitude (east is positive)
+    tz : int
+        number of timezones west of UTC (if date not in UTC)
```

Comment:
@klapo - Do you use the `tz` feature?  Above you say that `time` must be in UTC?  I'm thinking this should be removed and we should allow the `time` index to include a timezone.  Pandas has a normalization feature.